### DISCUSSION TITLE
info video

### DISCUSSION THREAD

**Discussion by marcio23** [smeisner/smart-thermostat#36] [Q&A] [answered]
In the meantime, I congratulate you, it's a great project and I would like to make it happen.
Do you have a video or photos to be able to see the real images of the various menus visible on the display?

**Comment by smeisner** [comment]
I do not, but I can easily take some and share. The images in the README are certainly dated, but they are close. I am not a graphics designer, so any improvements to the UI would be appreciated!

**Comment by marcio23** [comment]
Hi Steven, my name is Marco and I am Italian from Lecce. The project is very beautiful and I would like to make it happen.
Does everything work today, or is it not ready yet?
________________________________
Da: Steve Meisner ***@***.***>
Inviato: lunedì 8 gennaio 2024 15:53
A: smeisner/smart-thermostat ***@***.***>
Cc: marcio23 ***@***.***>; Author ***@***.***>
Oggetto: Re: [smeisner/smart-thermostat] info video (Discussion #36)


I do not, but I can easily take some and share. The images in the README are certainly dated, but they are close. I am not a graphics designer, so any improvements to the UI would be appreciated!

**Comment by marcio23** [comment]
I would add that I have never seen such a refined and cutting-edge project..... truly compliments
________________________________
Da: marco caputo ***@***.***>
Inviato: martedì 9 gennaio 2024 17:06
A: smeisner/smart-thermostat ***@***.***>
Oggetto: R: [smeisner/smart-thermostat] info video (Discussion #36)

Hi Steven, my name is Marco and I am Italian from Lecce. The project is very beautiful and I would like to make it happen.
Does everything work today, or is it not ready yet?
________________________________
Da: Steve Meisner ***@***.***>
Inviato: lunedì 8 gennaio 2024 15:53
A: smeisner/smart-thermostat ***@***.***>
Cc: marcio23 ***@***.***>; Author ***@***.***>
Oggetto: Re: [smeisner/smart-thermostat] info video (Discussion #36)


I do not, but I can easily take some and share. The images in the README are certainly dated, but they are close. I am not a graphics designer, so any improvements to the UI would be appreciated!

**Comment by smeisner** [comment, marked as answer]
Pics of the thermostat in use (for my basement furnace):
With the USB cable (for programming and debug) and the serial cable (for logging):
![PXL_20240109_204650576](https://github.com/smeisner/smart-thermostat/assets/6977132/ed66d208-38d9-4637-bf6e-65be8e49e60e)

General pics of some of the menus:

Info screen:
![Info screen](https://github.com/smeisner/smart-thermostat/assets/6977132/642794ce-8984-460a-bca7-c43b227979e3)

Settings:
![Settings](https://github.com/smeisner/smart-thermostat/assets/6977132/adff711a-e29c-42ab-9c37-82bc32a32502)

Additional settings:
![PXL_20240109_204729551](https://github.com/smeisner/smart-thermostat/assets/6977132/87289a13-fc42-404f-84bf-2224870b9611)

Wifi SSID selection:
![PXL_20240109_204740433](https://github.com/smeisner/smart-thermostat/assets/6977132/08b0f661-ee38-45e3-b4fa-12b764b85c78)

Device name menu:
![PXL_20240109_204754800](https://github.com/smeisner/smart-thermostat/assets/6977132/b55acb03-3610-4d43-9410-d7d06d5ee31e)

MQTT config screen:
![PXL_20240109_204810782](https://github.com/smeisner/smart-thermostat/assets/6977132/3db55661-0c32-49f5-8f7e-2bc7007b1767)

**Reply by smeisner** [reply]
Thank you for the kind words!!

The unit in the pictures is actually in use and controlling a furnace in my home. I have had it running for ~3 months now. The thermostat is running and working very well, especially integrated into Home Assistant. 

I don't have the Matter code running yet. I was waiting for the ESP Matter SDK to get more refined.

The projects is close to a V1.0 release since it is reliably operating.

If you'd like to get more involved, email me at steve at meisners dot net.

**Comment by smeisner** [comment]
One more pic showing the unit with the touchscreen removed. The 3D printed case is an old one I use on my desk for test units.

![PXL_20240109_212055270](https://github.com/smeisner/smart-thermostat/assets/6977132/1833913c-88d4-4584-bda8-65ef11ffb483)
